src/containers/StructurePage/plannedResource/PlannedResourceForm.tsx

```diff
@@ -305,13 +314,32 @@ const PlannedResourceForm = ({ articleType, node, onClose, userData }: Props) =>
           <StyledFormikField name="relevance">
             {({ field }: FieldProps) => (
               <SwitchWrapper>
-                <StyledLabel htmlFor="toggleRelevanceId">{t('taxonomy.resourceType')}</StyledLabel>
                 <RelevanceOption
                   relevanceId={field.value}
                   onChange={(id: string) => {
                     field.onChange({ target: { name: field.name, value: id } });
                   }}
                 />
+                <StyledLabel htmlFor="toggleRelevanceId">{t('taxonomy.resourceType')}</StyledLabel>
+              </SwitchWrapper>
+            )}
+          </StyledFormikField>
+          <StyledFormikField name="prioritized">
+            {({ field, form }: FieldProps) => (
+              <SwitchWrapper>
+                <StyledToggleSwitch>
+                  <StyledSwitch
+                    checked={prioritized}
+                    label=""
+                    id="togglePrioritized"
+                    onChange={(prioritized: boolean) => {
+                      form.setFieldValue(field.name, prioritized);
+                      setPrioritized(prioritized);
+                    }}
+                    thumbCharacter="P"
+                  />
+                </StyledToggleSwitch>
+                <StyledLabel htmlFor="togglePrioritized">{t('taxonomy.addPriority')}</StyledLabel>
```

Comment:
Det funker ikkje å klikke på label for å trigge toggle. Det hadde eg trudd skulle virke, men er kanskje ikkje meininga?

Reply:
Det er fordi den innebygde labelen ikke blir brukt, gjorde det sånn for å endre rekkefølge på toggle/label, ikke den beste løsningen men.. Kunne eventuelt oppdatert Switch-komponent til å støtte label foran og bak basert på en prop..